service/entrypoint.sh

```diff
@@ -7,7 +7,8 @@
 set -euo pipefail
 
 # For logback tagging (allow for local fallback)
-export HOST_IP=$(wget -qO- http://169.254.169.254/latest/meta-data/local-ipv4 || printf 'UNAVAILABLE')
+HOST_IP=$(curl --fail -s http://169.254.169.254/latest/meta-data/local-ipv4 || printf 'UNAVAILABLE')
+export HOST_IP
```

Comment:
Is separate export required as we use "||"? We might want to use -sSf also o get the error message to stderr for debugging purposes.

Reply:
If the separate export is because linting then ignore me.

Reply:
It was indeed because of the linter but you make a good point, I'll fix these.